build.py

```diff
@@ -464,42 +566,74 @@ def exclude_horizontal(item):
 
 if __name__ == "__main__":
   try:
+    closure_dir = CLOSURE_DIR_NPM
+    closure_root = CLOSURE_ROOT_NPM
+    closure_library = CLOSURE_LIBRARY_NPM
+    closure_compiler = CLOSURE_COMPILER_NPM
+
+    # Load calcdeps from the local library
     calcdeps = import_path(os.path.join(
```

Comment:
Why do this before you validate the compiler you're using?

Reply:
The order does not really matter. We need to check that the calcdeps module imports and that the compiler does what we expect. If either calcdeps fails to import or the compiler fails to give the expected output the build cannot be done locally.